backend/src/controllers/comparisons/comparisons.controller.ts

```diff
@@ -91,13 +90,13 @@ const getActiveComparisonCars = async (
 };
 
 const getActiveComparisonStatus = async (
-  req: AuthRequest,
+  req: Request,
   res: Response,
   next: NextFunction,
 ): Promise<void> => {
   try {
     const comparison = await comparisonsService.getActiveComparisonStatus(
-      req.tokenPayload?.sub as string,
+      req.tokenPayload.sub,
     );
```

Comment:
In controllers, we should use typed requests with typed body!. Untyped `Request` can be used only in middelwares

Reply:
Ouch, but now we don't have any difference between them